### PR TITLE
Angularlytics tracking

### PR DESCRIPTION
**Pull request by sparshithp**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/review.ninja/pull/1005" target="_blank">ReviewNinja

File: src/client/directives/templates/onboard.html [hunk 1]
```diff
@@ -21,6 +21,6 @@
     <a ui-sref="home({addrepo: true})" style="text-decoration:underline"><strong>Add repo</strong></a>.
   </div>
 
-  <button type="button" class="close" style="top:-20px;" data-dismiss="alert" ng-click="dismiss('taskbar2')">&times</button>
+  <button type="button" analytics-on="click" analytics-event="Onboard alert dismiss" analytics-category="Basic" class="close" style="top:-20px;" data-dismiss="alert" ng-click="dismiss('taskbar2')">&times</button>
```
Comment (dfarr) [inline src/client/directives/templates/onboard.html]: Let's call this one `Onboarding dismissed` !fix

File: src/client/templates/reviewItem.html [hunk 2]
```diff
@@ -108,7 +108,15 @@
                     </li>
                   </ul>
                 </div>
-                <button type="submit" class="btn btn-primary ob-leavecomment"  ng-disabled="!reviewComment">Comment</button>
+                <button 
+                  type="submit"  
+                  analytics-on="click" 
+                  analytics-event="Line Comment added"
```
Comment (dfarr) [inline src/client/templates/reviewItem.html]: `Review comment created` !fix

**Comment by dfarr** [comment]
Okay we just need to add events for
- [ ] Assign PR
- [ ] Onboarding completed
- [ ] Conversation comment created

Don't forget to sign the CLA :smile:

File: src/client/directives/templates/onboard.html (hunk 1 above)
Comment (dfarr) [inline src/client/directives/templates/onboard.html]: !fixed

File: src/client/templates/reviewItem.html (hunk 2 above)
Comment (dfarr) [inline src/client/templates/reviewItem.html]: !fixed

**Pull request merged**